bubuku/cli.py

```diff
@@ -227,5 +227,30 @@ def show_stats():
         _print_table(table)
 
 
+@cli.group(name='validate', help='Validates internal structures of kafka/zk')
+def validate():
+    pass
+
+
+@validate.command('replication', help='Checks zk structure for correct replica list')
```

Comment:
As there were no particular requirements for that feature, I think we should keep it as simple as possible, and just check for validity, once rebalance is finished (can be observed on the metrics)